index.html

```diff
@@ -0,0 +1,474 @@
+                </div>
+                <p>Reaching back across 13.8 billion years to the first sliver of cosmic time with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called gravitational waves — the signature of a universe being wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
+                <p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
+                <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.</p>
+                <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+            </section>
+            <section class="advert"></section>
+            <section class="text">
+                <h3>‘As Big as It Gets’</h3>
+                <p>In our own universe, it would serve as a window into the forces operating at energies forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s theory of general relativity, should ruffle space-time.</p>
+                <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+                <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”
+                <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+                <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+            </section>
+            <section class="feature">
+                <h3>The Theory of Inflation</h3>
```

Comment:
## Improve the `Theory of Inflation` Section
- [ ] Images for this section are missing as evident in the contrasting images below. Kindly add them. 

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/francisuloko/new-york-times-clone/add-article/index.html)|
|:--:|:--:|
|![theory-of-inflation](https://user-images.githubusercontent.com/22345961/108377052-ac444600-7214-11eb-86b0-5bdf8bb213f5.png)|![theory-of-inflation-clone](https://user-images.githubusercontent.com/22345961/108377068-b23a2700-7214-11eb-8eb4-34677655d76c.png)|

Reply:
Thank you for your feedback Christian, your guide was really helpful. I have applied the recommended changes. Although on my local machine the image displays correctly,  but  I have since updated the file path this time.